slackcollector/collector.py

```diff
@@ -148,8 +149,12 @@ def print_out(self, message, type='INFO'):
     # Create a new Collector instance
     # and pass the configuration as a param
     collector_inst = Collector()
+
+    dir = os.path.dirname(__file__)
```

Comment:
I have a problem with those, I need them be inside the Collector class and inside the `load_config()` method. For unit testing purposes, for exception handling purposes and separation of concerns.